rosdep/base.yaml

```diff
@@ -5620,6 +5620,9 @@ yamllint:
     xenial: [yamllint]
     yakkety: [yamllint]
     zesty: [yamllint]
+yapf:
+  ubuntu:
+    bionic: [python-yapf]
```

Comment:
So, there's actually an interesting thing here.  According to packages.ubuntu.com, there are actually separate packages: `yapf` provides the Python2 command-line utility `/usr/bin/yapf`, while the package `python-yapf` provides the library.  The situation is similar for python3.  If you want this key to be the command-line utility, then we should use `yapf` as the key name, and `yapf` as the package name.  If you want this key to be the library, then we should use `python-yapf` as the key name, and `python-yapf` as the package.  That way, we can have another set of keys for Python3 as well.

Reply:
Oh, I see. I changed it to the command-line utility. Also added the entry for ArchLinux. But couldn't find the package for Fedora 31.

Reply:
Awesome, thanks!